awx/main/management/commands/inventory_import.py

```diff
@@ -347,7 +347,12 @@ def _get_enabled(self, from_dict, default=None):
                     enabled = bool(unicode(enabled_value) == unicode(enabled))
                 else:
                     enabled = bool(enabled)
-        return enabled
+        if enabled is default:
+            return False
```

Comment:
Actually, maybe this should be `True` so that imports without any enabled var or value will import all hosts as enabled?

Reply:
I think returning `True` here makes sense.